### ISSUE TITLE
cors anywhere blocks https sites

### ISSUE THREAD

**Issue by valleyofdawn** [IsraelHikingMap/Site#1058]
## Bug
This external resource:
https://connecttile.garmin.com/ROAD_CYCLING/{z}/{x}/{y}.png
was not visible as an external source.

The error message in the console was:

Access to fetch at 'https://connecttile.garmin.com/ROAD_CYCLING/13/4887/3324.png' from origin 'https://israelhiking.osm.org.il' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

This resource is https not http, It's visible in other custom-enabled maps and should not be blocked.
This seems to be a separate issue from the quota issue seen before.

**Comment by HarelM** [comment]
This is due to mapbox and garmin limitations. I do not want to make our server a proxy for all the tiles requests as it will load it for no reason.
You are welcome to open a request for garmin and/or close this issue.

**Comment by HarelM** [comment]
See here:
https://github.com/mapbox/mapbox-gl-js/issues/3009

**Comment by valleyofdawn** [comment]
1) I see no reason why Garmin should support display of their tiles by outside apps. 
2) What you are saying is that even with the https prefix this resource does not support CORS and is therefore blocked?

**Comment by HarelM** [comment]
Yes, HTTPS and CORS are two different things. A site, like ours, that is hosted on a secure server and is reachable only in HTTPS can and should present content that is server over HTTPS as well.
This is why HTTP sites do not work.
[CORS ](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) means that a site can share its resources with other sites or not - it's a server configuration. If this is not enabled the browser in some cases, like the above which is how Mapbox decided to implement this, will not present content from server when CORS is disabled.
In theory, we can redirect all the traffic to our server and request the resources using the server code which does not require CORS, but it will have a large toll on our server's traffic which I do want.

**Comment by HarelM** [comment]
The following list describes other proxies which we might be able to use that may not have a request limitation:
https://hackya.com/us/list-of-free-cors-proxy-server/
@valleyofdawn you can try and use any of them as a prefix to the address you have.
Closing this issue as it is basically a duplicate of #1028
